### DISCUSSION TITLE
Concurrent doubling of plot times - any ideas?

### DISCUSSION THREAD

**Discussion by kushmoke** [swar/Swar-Chia-Plot-Manager#212] [Q&A]
I've been trying a few different settings on my system to get the plotting process optimised, but for each iteration of a plot sequence, each stage gets doubled in completion time. As a demo i have below max for stage 1 = 1 so that there are only every 3 in stage 1 at a time (1 plotting to each of my 3 SSD's) and 4 cores/ 4gb ram. As you can see, after the first few have passed stage 1, then second round are taking twice the amount of time to plot, even though they are essentially the same process, this then doubles exponentially until it takes longer to plot than it would get a coin. I have a Ryzen 9 3950x, 64gb ram, 3 tmp 2tb 870QVO SSD's. Does anyone know of any threading issue/ is there a limit on what can be run? TIA!
![Capture](https://user-images.githubusercontent.com/83551420/117563602-e6e4ae80-b09e-11eb-89b0-585725452e9a.PNG)

**Comment by pmadison** [comment]
Your SSD uses a QLC cache, you should try to find some readings on QLC and TLC SSD caches, and briefly look over the performance section of this page: https://www.pcworld.com/article/3562829/samsung-ssd-870-qvo-review.html
